index.html

```diff
@@ -314,6 +314,10 @@
         specification assumes the use of this protocol; alternative protocols are expected to
         provide compatible semantics.
       </p>
+      <p>
+        The <dfn>Content-Encoding</dfn> HTTP header, described in section 3.1.2.2. of [[!RFC7231]],
```

Comment:
The RFC style guide references sections using "Section 3.1.2.2", capital on Section and no trailing period.